static/js/EN/EN.js

```diff
@@ -732,7 +733,7 @@ $("#chartSelect").change(function(){
 });
 
 function toggleUnits() {
-    var v, v_a, el;
+    var v, v_a; //, el;
```

Comment:
Remove the `//, el;`?

Reply:
el was not used in the code. I have reverted the change but there is no need to keep it since it is not used.